src/platform_impl/linux/common/xkb_state.rs

```diff
@@ -154,7 +154,7 @@ impl KbdState {
     /// `xkb_state_key_get_utf8`.
     fn make_string_with<F>(&mut self, mut f: F) -> Option<SmolStr>
     where
-        F: FnMut(*mut i8, usize) -> i32,
+        F: FnMut(*mut core::ffi::c_char, usize) -> i32,
```

Comment:
could you do full export at the top of the file?

Reply:
Its already imported, I just didn't check lol :sweat_smile: 

Just changed this to `c_char`